py-deploy/tldeploy/cli.py

```diff
@@ -143,14 +144,11 @@ def currencynetwork(
             "Prevent mediator interests is not necessary if custom interests are disabled."
         )
 
-    if expiration_date is not None and expiration_time is not None:
+    if expiration_date is not None and expiration_time != 0:
```

Comment:
Please have None as default. 
If you do it this way, someone could try to disable it with 0, but also accidentally provide a date

Reply:
Okay make sense. :thinking: 
So to get then `0` as default for the deployment if the user does not define it, if would do something like:

```python
if expiration_time is None:
    expiration_time = 0
```

I'm thinking about the perspective of the user. `None` is the "default" which translates to `0` if the user defines nothing. This is the same as he would specifically set this parameter to `0`. Do we want to extend the help text for that information? Or is this just clear, no expiration date no freezing?

Reply:
Only set it to 0 if both are None. Just use the condition that was there before you removed it.

Reply:
Yes, I would add a help for both time and date, that explains the default is no freezing.

Reply:
> This is the same as he would specifically set this parameter to 0

No it is not the same, this is why you should not do it this way. 
0 means he wants it to be 0, so we should give him an error if he also provided a contradicting date arg.

Reply:
Understood. I will do it this way. :+1: